site/static/md/reference/sdks.md

```diff
@@ -0,0 +1,61 @@
+# SDKs
+
+Datastar provides backend SDKs that simplify the process of generating [SSE events](/reference/sse_events) specific to Datastar.
+
+## Go
+
+[Go package](https://github.com/starfederation/datastar/tree/develop/sdk/go) for working with Datastar.  
+_Author: [Delaney Gillilan](https://github.com/delaneyj)_
+
+Examples:
+
+- [1 App 5 Stacks ported to Go+Templ+Datastar](https://github.com/delaneyj/1a5s-datastar)
+
+## PHP
+
+[PHP package](https://github.com/starfederation/datastar-php) for working with Datastar.  
+_Author: [Ben Croker](https://github.com/bencroker) ([PutYourLightsOn](https://putyourlightson.com/))_
+
+### Laravel
+
+[Laravel package](https://github.com/putyourlightson/laravel-datastar) for working with Datastar.
+
+### Craft CMS
+
+[Craft CMS plugin](https://putyourlightson.com/plugins/datastar) for working with Datastar.
+
+Examples:
+
+- [Craft Datastar Pokemon demo](https://github.com/khalwat/craft-datastar-pokemon-demo)
+
+## Python
+
+[Python package](https://github.com/starfederation/datastar/tree/develop/sdk/python) for working with Datastar, including support for Sanic, Django, Quart and FastAPI.  
```

Comment:
Maybe add a link to correct package ("datastar" was already taken on PyPI):
```md
The SDK is also available as PyPI package [datastar-py](https://pypi.org/project/datastar-py/).
```

Reply:
Added, thanks!